src/main/java/com/rest/hw/app/controller/FileController.java

```diff
@@ -0,0 +1,55 @@
+package com.rest.hw.app.controller;
+
+import com.rest.hw.app.entity.FileDTO;
+import com.rest.hw.app.service.FileService;
+import lombok.AllArgsConstructor;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+public class FileController {
+    FileService fileService;
+
+    @PostMapping(
+            value = "/upload_file",
+            consumes = MediaType.APPLICATION_JSON_VALUE,
+            produces = MediaType.APPLICATION_JSON_VALUE
+    )
+    public ResponseEntity<?> fileUpload(@RequestBody MultipartFile file) throws IOException {
+        fileService.uploadFile(file);
+        return new ResponseEntity<>(file, HttpStatus.OK);
+    }
+
+    @DeleteMapping(value = "/delete_file", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity deleteFile(@RequestBody MultipartFile file) throws IOException {
+        fileService.deleteFile(file);
+        return new ResponseEntity<>(file, HttpStatus.OK);
+    }
+
+    @PutMapping(value = "/rename_file", produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity changeFile(String nameNew, String nameOld) {
```

Comment:
Return void if you don't need to return any response body to the client.
Extract `String nameNew, String nameOld` into a class and use it as a request body